File: Level 2/JavaScript-Applications-April2015/Exam-Preparation/Products/Products-App/scripts/models/product.js
var Product = (function () {
    function Product(objectId, name, category, price, editable) {
        this.id = objectId;
        this.name = name;
        this.category = category;
        this.price = price;
        this.editable = editable;
    }

    return Product;
}());

File: Level 3/Database-Apps-July2015/07.JSON-Processing/01.ProductsShop/Models/Product.cs
namespace _01.ProductsShop.Models
{
    using System.Collections.Generic;

    public class Product
    {
        public Product()
        {
            this.Categories = new HashSet<Category>();
        }

        public int ProductId { get; set; }

        public string Name { get; set; }

        public decimal Price { get; set; }

        public virtual User Seller { get; set; }

        public virtual User Buyer { get; set; }

        public virtual ICollection<Category> Categories { get; set; }
    }
}